src/main/java/com/google/devtools/build/lib/skyframe/BzlmodRepoRuleFunction.java

```diff
@@ -148,7 +148,11 @@ public SkyValue compute(SkyKey skyKey, Environment env)
       return BzlmodRepoRuleValue.REPO_RULE_NOT_FOUND_VALUE;
     }
     RepoSpec extRepoSpec = extensionEval.getGeneratedRepoSpecs().get(internalRepo);
-    Package pkg = createRuleFromSpec(extRepoSpec, starlarkSemantics, env).getRule().getPackage();
+    BzlmodRepoRuleValue repoRuleValue = createRuleFromSpec(extRepoSpec, starlarkSemantics, env);
```

Comment:
huh. it seems that something got lost in the history here. We should actually just `return createRuleFromSpec(extRepoSpec, starlarkSemantics, env);` and be done with this :)

Reply:
Just made that change in the browser, will see how it goes :⁠-⁠D